src/components/Footer/Footer/Footer.tsx

```diff
@@ -0,0 +1,34 @@
+import {
+  WhiteFacebookIcon,
+  WhiteYoutubeIcon,
+  WhiteTwitterIcon,
+  WhiteInstagramIcon,
+} from "@/src/asset/";
+import styles from "./Footer.module.scss";
+import Link from "next/link";
+
+export default function Footer() {
+  return (
+    <footer className={styles.footer}>
+      <p className={styles.codeit}>©codeit - 2024</p>
+      <div className={styles.linkBox}>
+        <Link href="/">Privacy Policy</Link>
+        <Link href="/">FAQ</Link>
+      </div>
+      <ul className={styles.snsBox}>
+        <li>
+          <WhiteFacebookIcon />
+        </li>
```

Comment:
대략 이런식으로 표현되어야할 것 같네요!
```suggestion
        <li>
          <a href="..." target='_blank' rel="noreferrer noopenner">
            <WhiteFacebookIcon />
          </a>
        </li>
```

Reply:
이해한대로 말씀드려보면
icon이나 svg 파일을 컴포넌트처럼 사용하면 width, height, fill같은 보여지는 속성들만 prop으로 제어하고 그 외에는 재사용을 위해 컴포넌트 안에서 더 설정하지 않는것일까요?

Reply:
next 안에서 다른 컴포넌트로(내부요소) 이동할 시에는 Link태그를 사용하고 외부 링크로 연결되는 경우에는 a태그 사용이 적절할까요?

Reply:
> 이해한대로 말씀드려보면
> icon이나 svg 파일을 컴포넌트처럼 사용하면 width, height, fill같은 보여지는 속성들만 prop으로 제어하고 그 외에는 재사용을 위해 컴포넌트 안에서 더 설정하지 않는것일까요?

네~ 외부에서 변경할 수 있는 속성들만 props으로 제어한다는 느낌으로 봐주시면 될 것 같습니다!

> next 안에서 다른 컴포넌트로(내부요소) 이동할 시에는 Link태그를 사용하고 외부 링크로 연결되는 경우에는 a태그 사용이 적절할까요?

네! 저는 그렇게 생각합니다~